olm-catalog/serverless-operator/project.yaml

```diff
@@ -34,15 +34,15 @@ dependencies:
   net_istio_artifacts_branch: release-1.4
   maistra: 2.1.0
 
-  eventing: 1.4
+  eventing: knative-v1.4
   # eventing core midstream branch name
   eventing_artifacts_branch: release-v1.4
 
   eventing_kafka: 1.1.0
-  # eventing-kafka-broker promotion version without the `knative-v` prefix
-  eventing_kafka_broker: 1.4
-  # eventing-kafka-broker midstream branch name
-  eventing_kafka_broker_artifacts_branch: release-v1.4
+  # eventing-kafka-broker promotion tag
+  eventing_kafka_broker: f24550ef10c9deea57c2306db05d813406ee73b8 # tag corresponds to a build of release v1.4
```

Comment:
does _this_ require a manual "bump", if we want to ship something different? Yes, right?

Reply:
Yes. We want to make a conscious decision

Reply:
Doesn't this really slow us down?

Today, using `latest` we do get the updated images implicitly, now, when we want to address this - or update shipped software, we do have to constantly commit to this repo. 

Is this repo really the right repo to address this concern? 

I fear that this adds more unneeded load, or one use-cases: faster updates to "managed services"

Reply:
I think, when we discussed this, we agreed that we keep using "latest for the branch" (aka floating) but when downstream users need a specific fix/commit form ekb, we will update it, open a PR, run the CI tests, merge (so that we can rebuild index images, etc), and then revert to floating

Reply:
floating is a huge productivity factor. 

Which makes me wonder if we try to "fix/solve" the issue of "fast delivery", on a wrong level, or layer